src/webapp/components/current-call/overview/Overview.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+import { CtaButtons } from "./CtaButtons";
+import { glassColors } from "../../../pages/app/themes/dhis2.theme";
+import { CurrentStatus } from "./CurrentStatus";
+
+interface OverviewProps {
+    moduleName: string;
+}
+
+export const Overview: React.FC<OverviewProps> = ({ moduleName }) => {
+    const [screen, setScreen] = useState<string>("status");
```

Comment:
`screen` seems to have only 2 values, so it's better to narrow the type: `type Status = "status" | "error"`

Reply:
fix now applied: https://github.com/EyeSeeTea/glass-dev/commit/948ed4c0439c7dcd66fa0d621d2cb11f8154cb53